scripts/lib/ansible-executor.sh

```diff
@@ -109,4 +110,12 @@ echo "Executing:"
 echo "    $cmd $args"
 echo "in a docker container $ansiblecm_container..."
 
-docker exec -it $ansiblecm_container $cmd $args
+# Normally we want to use the -it flag, but with GitHub actions you will get an
+# error for 'not a tty device'. This is simply saying that the color formatting
+# and other interactive features of the terminal are not supported, so we do not
+# submit those flags if that is the case. 
+it_flags="-it"
+if ! [ -z ${github_action+x} ]; then it_flags=""; fi
```

Comment:
`-it` means "interactive" so it also breaks if there is any interaction required from user, ie prompt

This will break those 2 flows you mentioned recently that have prompts if ran from non-tty device.

Reply:
It doesn't break because `--syntax-check` unfortunately doesn't check included tasks, which are the tasks you are referring to.